deepctr_torch/layers/core.py

```diff
@@ -49,6 +51,10 @@ def __init__(self, inputs_dim, hidden_units, activation=F.relu, l2_reg=0, dropou
         if self.use_bn:
             self.bn = nn.ModuleList(
                 [nn.BatchNorm1d(hidden_units[i + 1]) for i in range(len(hidden_units) - 1)])
+        
+        self.activation_layers = nn.ModuleList(
+            [activation_layer(self.activation, hidden_units[i + 1]) for i in range(len(hidden_units) - 1)])
+
```

Comment:
Making the code less robust

Reply:
Yes~ I am going to complete the code here to support all kinds of built-in activation layers in `PyTorch`.